### PR TITLE
Feature/sendgrid email

### LINKED ISSUE #26: Set up: Email API (Sendgrid)
https://sendgrid.com/solutions/email-api/

### PR DESCRIPTION
**Pull request by WhoaNellie**
Set up sendgrid and basic route for sending email, functional but needs a lot of info from the appointment to not be so generic 
image

Also we should either set up a gmail to send email from or one from wherever we deploy(?)

### CODE REVIEW

File: server/routes/emailAPI.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+sgMail.setApiKey(process.env.SENDGRID_API_KEY);
+
+const router = express.Router();
+
+// Send confirmation email
+router.post("/api/email", auth, (req, res) => {
+  const msg = {
+    to: req.body.email,
+    // TODO: make project email/maybe also domain?
+    from: "uesttser@gmail.com",
```
Comment (sina-jamshidi) [inline server/routes/emailAPI.js:14]: yeah that could be a nice idea. Probably just stick to a gmail for now, we won't provide a free domain 🤣

**Review by sina-jamshidi — APPROVED** [review]
Nice! Thanks Nellie! Once availabilities are done we can get started on booking appointments and we can customize the emails more

File: server/routes/emailAPI.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    // TODO: add in name and/or email of user, what appountment/event type was booked
+    // TODO: write good copy
+    subject: "Someone Has Booked an Appointment with you",
+    text:
+      "Someone has booked your appointment. Log in to Calendapp to make any updates.",
```
Comment (mattchx) [inline server/routes/emailAPI.js:19]: Once the app is deployed we can add the address here!

**Review by mattchx — COMMENTED** [review]
Great job! I really like SendGrid, seems intiuative to work with!

**Pull request merged**